src/main/resources/model/instance.json

```diff
@@ -191,6 +191,10 @@
         "dateOfPublication": {
           "index": "source",
           "showInResponse": [ "search", "call-number-browse" ]
+        },
+        "place": {
+          "index": "standard",
+          "showInResponse": [ "search" ]
```

Comment:
I can't find a requirement that states that the field should be in the response.